httpbin/cmd/cmd.go

```diff
@@ -74,10 +74,20 @@ func mainImpl(args []string, getEnv func(string) string, getHostname func() (str
 		logger = slog.New(handler)
 	}
 
+	env := make(map[string]string)
+	for _, e := range os.Environ() {
+		v := strings.SplitN(e, "=", 2)
+		if !strings.HasPrefix(v[0], "HTTPBIN_") {
```

Comment:
Here's something I'd like to think about a little bit.  A random collection of thoughts:
- I'd be inclined to look for a more specific `HTTPBIN_ENV_` prefix, to further reinforce that these special env vars will show up in the response from the `/env` endpoint.
  - Especially in cases where the serivce is deployed in Kubernetes, which by default exposes a ton of details about other services in the cluster with `${SERVICE_NAME}_`-prefixed env vars. (This might be overly cautious, but I try to be extra conservative when making changes that stand a chance of exposing new information unexpectedly!)
- I'd also consider stripping the prefix from the keys returned by `/env`, so `HTTPBIN_ENV_FOO=1` would end up being returned as `{"FOO": "1"}`
  - Would this behavior be convenient/useful or too magical/confusing?

As the implementer and a potential user of this feature, what do you think about these options?

Reply:
I have just replaced the prefix as you suggested. The Kubernetes case is a very good reason to use a more specific prefix indeed.

> Would this behavior be convenient/useful or too magical/confusing?

I 'feel' that such magic could lead to confusion. 
What if an operator exports container with:

```
BAR=bar
FOO=foo
HTTPBIN_ENV_FOO=foo-or-goo
```

then it could lead to confusion like "Which `FOO` is listed by /env?"

I usually prefer when systems behave less magical and in more explicit way. So, I wouldn't mind seeing seemingly redundant `HTTPBIN_ENV_` prefix in the `/env` response.

Reply:
> (...) it could lead to confusion like "Which FOO is listed by /env?"

Or, we could make the env vars prefix configurable with `HTTPBIN_ENV_` as default stored in the constants

https://github.com/mccutchen/go-httpbin/blob/34a21a3ef81df5aa42185fcae2e5a608404ac6a0/httpbin/httpbin.go#L9-L14

Then, provide `--env-var-prefix` command line argument to let operators override it with desired prefix.

Reply:
Thanks for the input, I appreciate it!  For now, let's just keep this simple and use the `HTTPBIN_ENV_*` vars as we find them, without stripping the prefix.